app-connector-client/server/resources/api.yaml

```diff
@@ -1,430 +1,590 @@
   /events:
     post:
-      summary: 'Sends a new event to the connected Kyma cluster'
+      tags:
+        - "event"
+      summary: "Sends a new event to the connected Kyma cluster"
       requestBody:
-        description: 'Event payload'
+        description: "Event payload"
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/EventDetails'
+              $ref: "#/components/schemas/EventDetails"
       parameters:
-      - in: 'query'
-        name: 'localKyma'
-        description: 'deployment to local kyma'
-        required: true
-        schema:
-          type: 'boolean'
+        - in: "query"
+          name: "localKyma"
+          description: "deployment to local kyma"
+          required: true
+          schema:
+            type: "boolean"
       responses:
-        '200':
-          description: 'Successful operation'
+        "200":
+          description: "Successful operation"
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ServiceId'
-        '400':
-          description: 'Bad request'
+                $ref: "#/components/schemas/EventId"
+        "400":
+          description: "Bad request"
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/MetadataErrorResponse'
+                $ref: "#/components/schemas/ErrorResponse"
   /apis:
     post:
-      summary: 'Registers a new API on the connected Kyma cluster'
+      tags:
+        - "api"
+      summary: "Registers a new API on the connected Kyma cluster"
       requestBody:
-        description: 'API to be registered'
+        description: "API to be registered"
         required: true
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/ServiceDetails'
+              $ref: "#/components/schemas/ServiceDetails"
       parameters:
-      - in: 'query'
-        name: 'localKyma'
-        description: 'deployment to local kyma'
-        required: true
-        schema:
-          type: 'boolean'
+        - in: "query"
+          name: "localKyma"
+          description: "deployment to local kyma"
+          required: true
+          schema:
+            type: "boolean"
       responses:
-        '200':
-          description: 'Successful operation'
+        "200":
+          description: "Successful operation"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ServiceId"
+        "4XX":
+          description: "Error message returned by server"
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ServiceId'
-        '400':
-          description: 'Bad request'
+                $ref: "#/components/schemas/ErrorResponse"
+
+        "500":

```

Comment:
Either add a 500 everywhere or remove it everywhere. I vote for removal

Reply:
removed